cloudevents/extensions/bam.md

```diff
@@ -0,0 +1,129 @@
+# Business Activity Monitoring (BAM) Extension
+
+Business Activity Monitoring (BAM) was originally coined by analysts at
+Gartner, and refers to the aggregation, analysis, and presentation of
+real-time information about activities inside organizations, customers,
+and partners.
+
+The activity monitoring consists of a model of a business process,
+which can consists of multiple transactions (e.g. order, payment, invoice),
+and these transactions can have multiple steps. The technical processing
+represented by a transaction instance `bamtxid` is then correlated with
+the steps in those transactions of the business process.
+
+This extension defines attributes that can be included within a CloudEvent
+to describe the business activity that the event is associated with.
+
+Producers and consumers are free to define an out-of-band agreement on the
+semantic meaning, or valid values, for the attribute.
+
+## Notational Conventions
+
+As with the main [CloudEvents specification](../spec.md), the key words "MUST",
+"MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+However, the scope of these key words is limited to when this extension is
+used. For example, an attribute being marked as "REQUIRED" does not mean
+it needs to be in all CloudEvents, rather it needs to be included only when
+this extension is being used.
+
+## Attributes
+
+### bamtxid
```

Comment:
This is shortening the name further than I had thought on the call. As such, would it make sense to have it expanded for readability such as:
```
### bamtxid (Business Transaction ID)
```
Note: neither the spec of other extensions have this but those attributes are more readable. @duglin thoughts?

Reply:
I think this makes sense.

```markdown
### `bamtxid` (Business Transaction ID)
```

Would strike a balance between efficiency of the fields and a good documentation. I think the convention on not using CamelCase, for the obvious reason of compatibility between platforms, this strikes the balance of actually make the long intention readable by shortening it. But on the other hand make it clear in the docs what is should be used for.

Reply:
I agree that some of the names I came up with are a bit short and I'd be ok with making them a bit longer - I'm just not that creative :-). But if folks are ok with these short names then yes I agree with adding the descriptive long names in parens.